CMSgov/cms-carts-seds
# Create base for new 2024 template

### Description

The shiny new 2024 template is here! Theres honestly not too many changes, but to list what has been done:

- A preamble has been added to the top of the state user homepage. It contains 3 paragraphs on information about CARTS
- A new base template and seed file were created for 2024 with the content updates including 1 new question in Section 5
- A feature flag has been created and used to toggle the 2024 release
- The <11 mask has been added to relevant questions for 2024
- Certain calculations (Section 3c Part 2 Table, Part 3 Calc and Tables, Part 4 Calc and Tables, Section 5 Part 3 Table and Part 4 Table) have been hidden in the print view as they contain direct answers to questions about children.

What is _not_ done:

- The template download will not work. Currently waiting for the PDF from the BO's

### Related ticket(s)


CMDCT-3945

---
### How to test

Fire up the local instance and sign in as a stateuser
See the new 2024 template is there on the homepage as well as the preamble
Open the 2024 template and check out some new wording. Section 5 has a particularly large amount of text edits
Make some edits and submit the report
Sign in as an admin
Checkout the 2024 template and see that its gucci.

---
### Pre-review checklist


---
### Pre-merge checklist


#### Review
